src/components/Home.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { dataList } from '../redux/home/home';
+import Tab from './Tab';
+import './Home.css';
+
+function Home() {
+  const listOfData = useSelector((state) => state.home);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!listOfData.length) {
+      dispatch(dataList());
+    }
+  }, [dispatch, listOfData.length]);
+  return (
+    <div className="main-container">
+      <div className="header">
+        <h1 className="home-title">Disney Characters</h1>
+      </div>
+      <div className="columns">
+        <div className="row">
+          {
+                       listOfData.map((item) => (
+                         <NavLink
+                           to={`details/${item.name}`}
+                           key={item._id}
+                         >
+                           <Tab
+                             name={item.name}
+                             films={item.films.length}
+                             imageUrl={item.imageUrl}
+                             _id={item._id}
+                             index={item.index}
+                             key={item.name}
+                           />
+                         </NavLink>
+                       ))
+                    }
+        </div>
+      </div>
```

Comment:
- [ ] according to your project requirement third point under Technical requirements, you need to add a filter component which will help user search and select any character of their choice, kindly add this component 
project requirement :point_down: 
![Screenshot from 2023-02-09 20-58-02](https://user-images.githubusercontent.com/66097371/217924461-cc4c3ae8-a328-4d58-9580-542096374fbc.png)

Reply:
Hi @VICTOR-CHUKWU, thank you very much for the feedback, would you mind to clarify, do you mean like adding a search bar?

Reply:
@Gdiazdiaz  exactly, a search bar to search for a character,

Reply:
Ok, I got it, thanks